pkg/api/dirs.go

```diff
@@ -63,6 +64,16 @@ func (s *Service) dirUploadHandler(
 	}
 	defer r.Body.Close()
 
+	rLevelNum, err := strconv.ParseUint(r.Header.Get(SwarmRedundancyLevelHeader), 10, 8)
```

Comment:
see how we read headers here https://github.com/ethersphere/bee/blob/master/pkg/api/bzz.go#L39

Reply:
removed and took value from caller after header parse.